### ISSUE TITLE
setting the allow_pickle parameter of PPSD.load_npz

### ISSUE THREAD

**Issue by meghansmiller** [ThomasLecocq/SeismoRMS#10]
I was having an issue with Line 10 in Step 5, which was resulting in this (shortened here) error:

ValueError: Loading PPSD results saved with ObsPy versions < 1.2 requires setting the allow_pickle parameter of PPSD.load_npz to True (needs numpy>=1.10).

So I changed Line 10 in Step 5 to:
            ppsds[mseedid].add_npz(fn, allow_pickle=True)

**Comment by claudiodsf** [comment]
Hi Meghan, thanks for reporting.

Out of curiosity, did you update ObsPy (from version < 1.2 to version 1.2) in between two runs?

**Comment by meghansmiller** [comment]
Hi Thomas and Claudio,

Thanks for the quick response.

I first tried to run the notebook in my existing environment as I was already using Obspy 1.2 and the other toolkits you list in the Notebook (these were updated about 2 months ago when I starting teaching this term … teaching 2nd year geologists about geophysics and an intro to python via binder).  I got the error I mentioned, so then I set up the covid environment you suggested just in case my versions were slightly different.  But I got the same error message.  I initially thought it was from Line 9, but then realized it was actually Line 10.

I have obspy 1.2.1 and numpy 1.18.1 if that is helpful.

Meghan

----------------------------
Dr. Meghan S. Miller
Associate Professor, Research School of Earth Sciences
The Australian National University
https://anu.zoom.us/j/7081672527

On Apr 8, 2020, at 6:49 PM, Claudio Satriano <notifications@github.com<mailto:notifications@github.com>> wrote:



Hi Meghan, thanks for reporting.

Out of curiosity, did you update ObsPy (from version < 1.2 to version 1.2) in between two runs?
